clubs/views.py

```diff
@@ -100,20 +102,21 @@ def perform_create(self, serializer):
             status=Club.PENDING
         )
         send_mail(
-            subject=email.CREATE_MAIL_SUBJECT,
-            message=email.CREATE_MAIL_BODY % {
+            subject=email.create_club.plaintext.subject.render(Context()),
+            message=email.create_club.plaintext.body.render(Context({
```

Comment:
oh weird, I thought I had put a comment on top!
Basically the context is meant to provide placeholder replacements (kinda like the `.format` method for strings) in the template and since it is empty here you don't need to pass it in...in  django 1.9...but this is django 1.7 which requires you to pass it in always